app/src/js/controllers/ImageController.js

```diff
@@ -47,6 +48,44 @@ angular.module('media_manager')
       CourseCache.current_image = CourseCache.images[ic.index];
       resetBreadcrumb();
     }
-  }
+  };
+
+  ic.newLabel = '';
+  ic.newValue = '';
+  ic.saveMetadata = function(label, value){
+    if(label !== undefined && value !== undefined){
```

Comment:
This is more of a style comment, but I think the only situation in which label and value are undefined is when those arguments aren't passed to the function. So in my mind, the intention is this:

```javascript
if(arguments.length == 0) {
    createMetadata();
} else if(arguments.length == 2) {
    updateMetadata();
}
```

But if that's the case, why not just call the relevant method from the angular template to make it explicit?